spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -33,13 +33,16 @@ def get_parser():
 
               - volume `[mm^3]`: volume of the lesion
               - length `[mm]`: maximal length along the Superior-Inferior (SI) axis across all slices
+              - width `[mm]`: maximal width along the Anterior-Posterior (AP) axis across all slices
```

Comment:
Just for my own understanding: We use SI and AP for length/width because we're using a sagittal FOV (i.e. LR slices)?

If so, maybe this could be a helpful clarification:

```suggestion
              - length `[mm]`: maximal length along the Superior-Inferior (SI) axis across all sagittal slices of the lesion
              - width `[mm]`: maximal width along the Anterior-Posterior (AP) axis across all sagittal slices of the lesion
```

Reply:
Ah, but we actually use `axial` slices in `_measure_width` for computing the max AP width... same result, but minor inconsistency, hehe.

Reply:
> maybe this could be a helpful clarification

sounds good 👍🏻